scico/examples.py

```diff
@@ -291,3 +291,29 @@ def create_circular_phantom(
         img = img.at[dist_map < r].set(val)
 
     return img
+
+
+def spnoise(img: Array, frc: float, smn: float = 0.0, smx: float = 1.0) -> Array:
```

Comment:
what do smn and smx stand for? Perhaps longer, more descriptive names would be nicer.

Reply:
Agreed. Fixed.